examples/htlc.rs

```diff
@@ -0,0 +1,49 @@
+// Miniscript
+// Written in 2019 by
+//     Andrew Poelstra <apoelstra@wpsoftware.net>
```

Comment:
You're welcome to put your own name on this btw :)

Reply:
Ups, sorry :sweat_smile: 
I just copy-pasted another example file. Fixed!